third_party/pyth/price-service/src/rest.ts

```diff
@@ -135,13 +134,16 @@ export class RestAPI {
         ids: Joi.array()
           .items(Joi.string().regex(/^(0x)?[a-f0-9]{64}$/))
           .required(),
+        verbose: Joi.boolean(),
       }).required(),
     };
     app.get(
       "/api/latest_price_feeds",
       validate(latestPriceFeedsInputSchema),
       (req: Request, res: Response) => {
         let priceIds = req.query.ids as string[];
+        // verbose is optional, default to false
```

Comment:
it could be default to undefined too 🤔

Reply:
if it's undefined it will default to false since it's doing the check in the next line 🤔